stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -117,7 +118,9 @@
         ;;  previous stack-stx calls, or prior to an extend)
         reward-set-indexes: (list 12 uint),
         ;; principal of the delegate, if stacker has delegated
-        delegated-to: (optional principal)
+        delegated-to: (optional principal),
+        ;; signing key for Nakamoto+, only 'none' when delegated & before delegate calls 'stack-aggregation-commit-indexed
+        signing-key: (optional (buff 33))
```

Comment:
It seems like this could just be `(buff 33)` rather than an `(optional (buff 33))`: the delegate sets this value in `delegate-stack-stx`, which is also the first time the stacker-state is instantiated for the delegator.

Reply:
Does it make sense to allow delegators enforce a signing key? If yes, should it be stored in a separate map?

Reply:
Sorry, I meant the user/pool member. My question does not make sense here.

Reply:
I agree, that stack-aggregation-commit can't change the signing key.